README.md

```diff
@@ -170,12 +170,32 @@ You may also see any lint errors in the console.
 
 ## 👥 Authors <a name="authors"></a>
 
-👤👤 **Fisayo Michael Animashaun**
+👤 **Fisayo Michael Animashaun**
```

Comment:
- [ ] This link does not work:

![image](https://github.com/fmanimashaun/book-car/assets/57421823/a6f9ec95-3499-49e3-b2d6-b1129dcdca96)


![image](https://github.com/fmanimashaun/book-car/assets/57421823/666ceaa9-3628-431c-a2b4-ca780850c230)

Kindly correct the link. 👍